py_v_sdk/api.py

```diff
@@ -315,6 +315,17 @@ async def get_tok_id(self, ctrt_id: str, tok_idx: int) -> Dict[str, Any]:
         """
         return await self._get(f"/contractId/{ctrt_id}/tokenIndex/{tok_idx}")
 
+    async def get_tok_info(self, tok_id: str) -> Dict[str, Any]:
+        """
+        get_tok_info gets the info of the given token.
+        Args:
+            tok_id (str): the token ID.
+
+        Returns:
+            Dict[str, Any]: The response.
+        """
+        return await self._get(f"/tokenInfo/{tok_id}")
```

Comment:
As this method exists in the `develop` branch already, can rebase this PR branch with the latest `develop` branch and remove this method.